packages/CollapsibleChecklists/stories/examples/Basic.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import InfoIcon from "@paprika/icon/lib/InfoCircle";
+import Spinner from "../../../Spinner/src/index";
+import CollapsibleChecklists from "../../src/index";
+
+const ExampleStory = () => {
+  const handleHover = () => {
+    console.log("Show a popover as to why its disabled");
+  };
+
+  const [sportsData, setSportsData] = React.useState([
+    {
+      state: "California",
+      sports: [
+        {
+          title: <span>Basketball</span>,
+          teams: [
+            { isChecked: true, isDisabled: false, name: "Golden State Warriors" },
+            { isChecked: false, isDisabled: false, name: "LA Lakers" },
+            { isChecked: false, isDisabled: true, name: "LA Clippers" },
+            { isChecked: false, isDisabled: false, name: "Sacramento Kings" },
+          ],
+        },
+        {
+          title: "Hockey",
+          teams: [
+            { isChecked: true, isDisabled: false, name: "Anaheim Ducks" },
+            { isChecked: true, isDisabled: false, name: "LA Kings" },
+            { isChecked: true, isDisabled: false, name: "San Jose Sharks" },
+          ],
+        },
+        {
+          title: (
+            <span>
+              Baseball
+              <span
+                onFocus={() => {}}
+                onMouseOver={handleHover}
+                style={{ paddingLeft: "4px", color: "silver", cursor: "pointer" }}
+              >
+                <InfoIcon />
+              </span>
+            </span>
+          ),
+          teams: [
+            { isChecked: false, isDisabled: false, name: "Anaheim Angels" },
+            { isChecked: false, isDisabled: false, name: "LA Dodgers" },
+            { isChecked: false, isDisabled: false, name: "Oakland Athletics" },
+            { isChecked: false, isDisabled: false, name: "San Diego Padres" },
+            { isChecked: true, isDisabled: false, name: "San Francisco Giants" },
+          ],
+        },
+      ],
+    },
+    {
+      state: "Ohio",
+      sports: [
+        {
+          title: <span>Football (expand to retrieve from API)</span>,
+          teams: [
+            // This will later be retrieved from an API on expand
+          ],
+        },
+      ],
+    },
+  ]);
+
+  const handleExpand = () => {
+    setTimeout(() => {
+      const newSportsData = sportsData.slice(0);
+      newSportsData[1].sports[0].teams = [
+        { isChecked: false, isDisabled: false, name: "Cincinnati Bengals" },
+        { isChecked: false, isDisabled: false, name: "Cleveland Browns" },
+      ];
+      setSportsData(newSportsData);
+    }, 2000);
```

Comment:
create a static variable for this number eg `const EXPAND_DURATION=2000` 🤷‍♂

Reply:
It's just a story; think that is necessary?

Reply:
🤷‍♂